server/workers/userCreated.js

```diff
@@ -0,0 +1,28 @@
+import {connect} from 'src/db'
+import {findContactByEmails, addIdmToCrm} from 'src/server/actions/crm'
+import createMember from 'src/server/actions/createMember'
+
+const r = connect()
+
+export function start() {
+  const jobService = require('src/server/services/jobService')
+  jobService.processJobs('userCreated', processUserCreated)
+}
+
+export async function processUserCreated(idmUser) {
+  try {
+    const contactVid = (await findContactByEmails(idmUser.emails)).vid
```

Comment:
Should validate that contact is found before accessing `vid` prop. Otherwise, `TypeError`. Just throw an error with any reasonable messaging.